beacon_node/network/src/router/mod.rs

```diff
@@ -247,6 +247,8 @@ impl<T: BeaconChainTypes> Router<T> {
                 self.processor
                     .on_attester_slashing_gossip(id, peer_id, attester_slashing);
             }
+            PubsubMessage::SignedContributionAndProof(_contribution_and_proof) => unimplemented!(),
+            PubsubMessage::SyncCommitteeSignature(_sync_committtee) => unimplemented!(),
```

Comment:
these are actually unimplemented?

Reply:
Haven't implemented these yet. 
Need to build on top of https://github.com/sigp/lighthouse/pull/2321 . Have started some work on a local branch, but it's not complete yet.